util/libeay.num

```diff
@@ -4370,7 +4370,7 @@ DH_compute_key_padded                   4732	EXIST::FUNCTION:DH
 ECDSA_METHOD_set_sign                   4733	EXIST::FUNCTION:ECDSA
 CMS_RecipientEncryptedKey_cert_cmp      4734	EXIST:!VMS:FUNCTION:CMS
 CMS_RecipEncryptedKey_cert_cmp          4734	EXIST:VMS:FUNCTION:CMS
-DH_KDF_X9_42                            4735	EXIST::FUNCTION:DH
+DH_KDF_X9_42                            4735	EXIST::FUNCTION:CMS,DH
```

Comment:
> I can't see how that's our responsibility

And the question was if it *would* benefit users *if* it was [at least partially] our responsibility. Indeed, why isn't it? Why wouldn't it be impossible to omit something mid-series if there was *legitimate* need for that? For example, yes, it would be impossible to add no-rsa (not even mid-series) because you might need to verify stored signatures, but why wouldn't it be impossible to disable some less critical things without having to recompile application? [Again, provided that application pays attention to failure codes.] Yes, one can argue that there no legitimate need to add no-cms, but question is more general than this particular case... Yes, you can argue that this is not the place to bring this up, but there is no place at all...